app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
Make `skills` instance's attribute.
And then in child classes append skills to this attribute.